### PR TITLE
Put some haha funnies in the FightMessages

### PR DESCRIPTION
**Pull request by WaterGame2023**
Well we talked mentioned this in PNW Discord, so I did it

### CODE REVIEW

File: src/main/java/com/wylxbot/wylx/Commands/Fight/Util/FightMessages.java [hunk 1]
```diff
@@ -41,6 +41,16 @@ public class FightMessages {
         // Responses below by Macro (2898)
         "{p2} was forced to re-machine {r} parts by {p1}",
         "{p1} broke {p2}'s SD card",
+        // Responses below by WaterGame2023
+        "{p2} was told to use windows by {p1}",
+        "{p2} was forced to update windows by {p1}",
+        "{p1} used rm -rf on {p2}",
+        "{p2} got knocked the Hab by {p1}",
+        "{p2} lost connection to the field courtesy of {p1}",
+        "{p2} was forced to use C++ for the robot code by {p1}',
```
Comment (1Revenger1) [inline src/main/java/com/wylxbot/wylx/Commands/Fight/Util/FightMessages.java:50]:
```suggestion
        "{p2} was forced to use C++ for their robot code by {p1}',
```

File: src/main/java/com/wylxbot/wylx/Commands/Fight/Util/FightMessages.java [hunk 2]
```diff
@@ -41,6 +41,16 @@ public class FightMessages {
         // Responses below by Macro (2898)
         "{p2} was forced to re-machine {r} parts by {p1}",
         "{p1} broke {p2}'s SD card",
+        // Responses below by WaterGame2023
+        "{p2} was told to use windows by {p1}",
+        "{p2} was forced to update windows by {p1}",
+        "{p1} used rm -rf on {p2}",
+        "{p2} got knocked the Hab by {p1}",
```
Comment (1Revenger1) [inline src/main/java/com/wylxbot/wylx/Commands/Fight/Util/FightMessages.java:48]: Check Grammer here, maybe meant "knocked off the hab"?

File: src/main/java/com/wylxbot/wylx/Commands/Fight/Util/FightMessages.java [hunk 3]
```diff
@@ -41,6 +41,16 @@ public class FightMessages {
...
+        "{p2} lost connection to the field courtesy of {p1}",
+        "{p2} was forced to use C++ for the robot code by {p1}',
+        "{p2} was told to touch grass by {p1}',
+        "{p2} DDoSed by {p1}',
```
Comment (1Revenger1) [inline src/main/java/com/wylxbot/wylx/Commands/Fight/Util/FightMessages.java:52]:
```suggestion
        '{p1} DDoSed {p2}',
```

File: src/main/java/com/wylxbot/wylx/Commands/Fight/Util/FightMessages.java [hunk 4]
```diff
@@ -41,6 +41,16 @@ public class FightMessages {
         // Responses below by Macro (2898)
         "{p2} was forced to re-machine {r} parts by {p1}",
         "{p1} broke {p2}'s SD card",
+        // Responses below by WaterGame2023
+        "{p2} was told to use windows by {p1}",
```
Comment (1Revenger1) [inline src/main/java/com/wylxbot/wylx/Commands/Fight/Util/FightMessages.java:45]:
```suggestion
        "{p2} was told to use Windows by {p1}",
```

File: src/main/java/com/wylxbot/wylx/Commands/Fight/Util/FightMessages.java [hunk 5]
```diff
@@ -41,6 +41,16 @@ public class FightMessages {
...
+        "{p2} was forced to update windows by {p1}",
```
Comment (1Revenger1) [inline src/main/java/com/wylxbot/wylx/Commands/Fight/Util/FightMessages.java:46]:
```suggestion
        "{p2} was forced to update Windows by {p1}",
```

**Review by 1Revenger1 — CHANGES_REQUESTED** [review]

**Review by WaterGame2023 — COMMENTED** [review]
Added the suggestions.  Thank you

**Review by 1Revenger1 — APPROVED** [review]

**Pull request merged**